pkg/utils/k8sutil.go

```diff
@@ -711,7 +711,6 @@ func EnsureFuncService(client kubernetes.Interface, funcObj *kubelessApi.Functio
 		newSvc.ObjectMeta.Labels = funcObj.ObjectMeta.Labels
 		newSvc.ObjectMeta.OwnerReferences = or
 		newSvc.Spec.Ports = svc.Spec.Ports
-		newSvc.Spec.Selector = svc.Spec.Selector
```

Comment:
Wondering if this is causing any problem? Function labels can be changed (though cli may prevent it), in which case service selector may continue to use old selector.